paging-runtime-composeui/src/androidMain/kotlin/app/cash/paging/createMainDispatcher.kt

```diff
@@ -0,0 +1,6 @@
+package app.cash.paging
+
+import kotlinx.coroutines.Dispatchers
+import kotlin.coroutines.CoroutineContext
+
+actual fun createMainDispatcher(): CoroutineContext = Dispatchers.Main
```

Comment:
I may not need these. However, [Dispatchers.Main](https://github.com/androidx/androidx/blob/androidx-main/paging/paging-compose/src/main/java/androidx/paging/compose/LazyPagingItems.kt#L65) is hard-coded and injected into the [PagingDataDiffer](https://github.com/androidx/androidx/blob/androidx-main/paging/paging-compose/src/main/java/androidx/paging/compose/LazyPagingItems.kt#L105). I _think_ I will need to have this be platform specific, according to the [docs](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-dispatchers/-main.html), where a different artifact should be used for android vs jvm. Unless I am reading it wrong...